### ISSUE TITLE
Poor detection for a dense crowd of pedestrians with downfacing Kinect

### ISSUE THREAD

**Issue by zchenpds** [OpenPTrack/open_ptrack#100]
Wondering if it's possible to adapt the ground-based people detector to the case of Kinect sensors facing downward from a height of around 5 meters above the ground, under the assumption that the ground is free of protruding objects other than pedestrians. A similar closed source software to achieve this can be found [here](http://www.fz-juelich.de/ias/jsc/EN/Research/ModellingSimulation/CivilSecurityTraffic/PedestrianDynamics/Activities/petrack/petrackNode.html). 
Any suggestion as to what to refer to or how to modify the code would be appreciated.

**Comment by nanodust** [comment]
OPT was (thusfar) explicitly designed to NOT do top-down.
the person detection classifiers are not trained on top-down. they need the
torso.
see the paper listed here
<http://openptrack.org/overview-how-openptrack-works/> for algorithm
overview.

TLDR:
Overhead tracking w/ OPT is not a simple change; yet if you've trained
haars cascade classifiers in the past (or want to learn) you're welcome to
do so, and contribute back to the project :)

that said, if you're just looking for heads, top-down, that's a pretty
simple algorithm (blob tracking) and openPtrack is overkill for what you're
after. don't even need kinects, any camera will do.

http://ccv.nuigroup.com for example (which is a nice app / wrapper around
opencv blob tracking)

**Comment by zchenpds** [comment]
Thanks so much, Mr.  @nanodust . 

I am very new to this field and sorry I didn't make myself clear. We want to get rid of the markers on people's heads. So I am thinking of overhead people detection and tracking from point clouds. Since as you said it's not a simple change, and detection and tracking are not our focus, we may let it go later.
